src/components/Uploader/index.js

```diff
@@ -38,7 +39,10 @@ export const UploadError = ({errorMessage}) => (
   <div className={`${style.text} ${style.error}`}>{errorMessage}</div>
 )
 
-export const Uploader = ({method, onImageSelected, uploading, noDocument}) => (
+//TODO move to react instead of preact, since preact has issues handling pure components
+//IF this component is exported as pure,
+//some components like Camera will not have componentWillUnmount called
```

Comment:
I generally find that when putting in comments like these, it's helpful to add a link to the issue tracking the problem

That way you can have more detail, have an issue for tracking, and also have something to grep for easily when the issue is resolved